extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js

```diff
@@ -251,10 +251,17 @@ export class AmpRecaptchaService {
    */
   getRecaptchaFrameSrc_() {
     if (getMode().localDev || getMode().test) {
-      return ampToolboxCacheUrl.createCurlsSubdomain(this.win_.location.href)
+      let winLocation = this.win_.location;
+      if (this.win_.testLocation) {
```

Comment:
When will `this.win_.testLocation` be set?

Reply:
It is set in: https://github.com/ampproject/amphtml/blob/master/testing/iframe.js#L100

😄  Let me know what you think!

Reply:
I'm wondering what's the real value of `this.win_.location` and `this.win_.testLocation` in test mode. Should them both be `http://localhost:9876`?